From 92019e3d7f1859d52d6a74fbd705d73ee0446ebf Mon Sep 17 00:00:00 2001
From: "Ch1llb0/skin.osmc" <36623079+Ch1llb0@users.noreply.github.com>
Date: Wed, 20 Mar 2019 20:41:59 +0100
Subject: [PATCH] Skin settings bug fix

Custom_Skin_Helper_Service.xml:
- fix textbox label

Custom_Skin_Helper_Skin_Backup_Open.xml:
- fix textbox label

Custom_Skin_Helper_Skin_Backup_Settings.xml:
- fix textbox label

Custom_Skin_Helper_Service_Widgets.xml:
- fix textbox label

SkinSettings.xml:
- fix layout of add-ons section
- move supported label
- comment out Nextup Notification add-on (it's marked as broken)
---
 ...m_Skin_Helper_Service_Skin_Backup_Open.xml |  2 +-
 ...in_Helper_Service_Skin_Backup_Settings.xml |  2 +-
 16x9/Custom_Skin_Helper_Service_Widgets.xml   |  2 +-
 16x9/SkinSettings.xml                         | 50 ++++++++-----------
 4 files changed, 24 insertions(+), 32 deletions(-)

diff --git a/16x9/Custom_Skin_Helper_Service_Skin_Backup_Open.xml b/16x9/Custom_Skin_Helper_Service_Skin_Backup_Open.xml
index 0dcb975..ddc3285 100644
--- a/16x9/Custom_Skin_Helper_Service_Skin_Backup_Open.xml
+++ b/16x9/Custom_Skin_Helper_Service_Skin_Backup_Open.xml
@@ -51,7 +51,7 @@
 				<font>Font36</font>
 				<autoscroll>False</autoscroll>
 				<textcolor>$VAR[TextColor1]</textcolor>
-				<label>Please enable the following add-on under [I]Settings/Add-on browser/My add-ons/Context menus[/I] :[CR][LIGHT]Skin Helper Service Skin Backup[/LIGHT]</label>
+				<label>Please enable the following add-on under [I]Settings/Add-on browser/My add-ons/Program add-ons[/I] :[CR][LIGHT]Skin Helper Service Skin Backup[/LIGHT]</label>
 			</control>
 		</control>
 
diff --git a/16x9/Custom_Skin_Helper_Service_Skin_Backup_Settings.xml b/16x9/Custom_Skin_Helper_Service_Skin_Backup_Settings.xml
index 1bf81db..2daf60e 100644
--- a/16x9/Custom_Skin_Helper_Service_Skin_Backup_Settings.xml
+++ b/16x9/Custom_Skin_Helper_Service_Skin_Backup_Settings.xml
@@ -51,7 +51,7 @@
 				<font>Font36</font>
 				<autoscroll>False</autoscroll>
 				<textcolor>$VAR[TextColor1]</textcolor>
-				<label>Please enable the following add-on under [I]Settings/Add-on browser/My add-ons/Context menus[/I] :[CR][LIGHT]Skin Helper Service Skin Backup[/LIGHT]</label>
+				<label>Please enable the following add-on under [I]Settings/Add-on browser/My add-ons/Program add-ons[/I] :[CR][LIGHT]Skin Helper Service Skin Backup[/LIGHT]</label>
 			</control>
 		</control>
 
diff --git a/16x9/Custom_Skin_Helper_Service_Widgets.xml b/16x9/Custom_Skin_Helper_Service_Widgets.xml
index 6631e80..0c64c00 100644
--- a/16x9/Custom_Skin_Helper_Service_Widgets.xml
+++ b/16x9/Custom_Skin_Helper_Service_Widgets.xml
@@ -51,7 +51,7 @@
 				<font>Font36</font>
 				<autoscroll>False</autoscroll>
 				<textcolor>$VAR[TextColor1]</textcolor>
-				<label>Please enable the following add-on under [I]Settings/Add-on browser/My add-ons/Context menus[/I] :[CR][LIGHT]Skin Helper Service Widgets[/LIGHT]</label>
+				<label>Please enable the following add-on under [I]Settings/Add-on browser/My add-ons/Music add-ons[/I] or under [I]Settings/Add-on browser/My add-ons/Video add-ons[/I] :[CR][LIGHT]Skin Helper Service Widgets[/LIGHT]</label>
 			</control>
 		</control>
 
diff --git a/16x9/SkinSettings.xml b/16x9/SkinSettings.xml
index cd6d474..40c3a84 100755
--- a/16x9/SkinSettings.xml
+++ b/16x9/SkinSettings.xml
@@ -718,9 +718,8 @@
 				<control type="button" id="202">
 					<width>1200</width>
 					<height>66</height>
-					<left>20</left>
 					<textwidth>1120</textwidth>
-					<label>$LOCALIZE[31079]</label>
+					<label> - $LOCALIZE[31079]</label>
 					<label2>$VAR[addon-skinshortcuts]</label2>
 					<onclick condition="!System.HasAddon(script.skinshortcuts) | System.HasAddon(script.skinshortcuts) + !String.IsEqual(System.AddonVersion(script.skinshortcuts),1.0.18) + !System.HasAddon(service.osmc.settings)">ActivateWindow(1103)</onclick>
 					<onclick condition="System.HasAddon(script.skinshortcuts) + System.HasAddon(service.osmc.settings) | System.HasAddon(script.skinshortcuts) + String.IsEqual(System.AddonVersion(script.skinshortcuts),1.0.18) + !System.HasAddon(service.osmc.settings)">Addon.OpenSettings(script.skinshortcuts)</onclick>
@@ -728,9 +727,8 @@
 				<control type="button" id="203">
 					<width>1200</width>
 					<height>66</height>
-					<left>20</left>
 					<textwidth>1120</textwidth>
-					<label>$LOCALIZE[31081]</label>
+					<label> - $LOCALIZE[31081]</label>
 					<label2>$VAR[addon-skinhelper]</label2>
 					<onclick condition="!System.HasAddon(script.skin.helper.service) | System.HasAddon(script.skin.helper.service) + !String.IsEqual(System.AddonVersion(script.skin.helper.service),1.1.24) + !System.HasAddon(service.osmc.settings)">ActivateWindow(1104)</onclick>
 					<onclick condition="System.HasAddon(script.skin.helper.service) + System.HasAddon(service.osmc.settings) | System.HasAddon(script.skin.helper.service) + String.IsEqual(System.AddonVersion(script.skin.helper.service),1.1.24) + !System.HasAddon(service.osmc.settings)">Addon.OpenSettings(script.skin.helper.service)</onclick>
@@ -747,9 +745,8 @@
 				<control type="button" id="205">
 					<width>1200</width>
 					<height>66</height>
-					<left>20</left>
 					<textwidth>1120</textwidth>
-					<label>$LOCALIZE[31106]</label>
+					<label> - $LOCALIZE[31106]</label>
 					<label2>$VAR[addon-skinhelperwidgets]</label2>
 					<onclick condition="!System.HasAddon(script.skin.helper.widgets) | System.HasAddon(script.skin.helper.widgets) + !String.IsEqual(System.AddonVersion(script.skin.helper.widgets),1.0.38) + !System.HasAddon(service.osmc.settings)">ActivateWindow(1105)</onclick>
 					<onclick condition="System.HasAddon(script.skin.helper.widgets) + System.HasAddon(service.osmc.settings) | System.HasAddon(script.skin.helper.widgets) + String.IsEqual(System.AddonVersion(script.skin.helper.widgets),1.0.38) + !System.HasAddon(service.osmc.settings)">Addon.OpenSettings(script.skin.helper.widgets)</onclick>
@@ -757,58 +754,53 @@
 				<control type="button" id="206">
 					<width>1200</width>
 					<height>66</height>
-					<left>20</left>
 					<textwidth>1120</textwidth>
-					<label>$LOCALIZE[31097]</label>
+					<label> - $LOCALIZE[31097]</label>
 					<label2>$VAR[addon-skinhelperbackup]</label2>
 					<onclick condition="!System.HasAddon(script.skin.helper.skinbackup) | System.HasAddon(script.skin.helper.skinbackup) + !String.IsEqual(System.AddonVersion(script.skin.helper.skinbackup),1.0.19) + !System.HasAddon(service.osmc.settings)">ActivateWindow(1106)</onclick>
 					<onclick condition="System.HasAddon(script.skin.helper.skinbackup) + System.HasAddon(service.osmc.settings) | System.HasAddon(script.skin.helper.skinbackup) + String.IsEqual(System.AddonVersion(script.skin.helper.skinbackup),1.0.19) + !System.HasAddon(service.osmc.settings)">Addon.OpenSettings(script.skin.helper.skinbackup)</onclick>
 				</control>
-				<control type="button" id="207">
+				<!-- Supported -->
+				<control type="label" id="207">
+					<width>1220</width>
+					<height>62</height>
+					<font>Font33</font>
+					<align>left</align>
+					<label>[I]$LOCALIZE[31084][/I]</label>
+					<textcolor>$VAR[TextColor2]</textcolor>
+				</control>
+				<control type="button" id="208">
 					<width>1200</width>
 					<height>66</height>
-					<left>20</left>
 					<textwidth>1120</textwidth>
-					<label>$LOCALIZE[31082]</label>
+					<label> - $LOCALIZE[31082]</label>
 					<label2>$VAR[addon-autocompletion]</label2>
 					<onclick condition="!System.HasAddon(plugin.program.autocompletion)">InstallAddon(plugin.program.autocompletion)</onclick>
 					<onclick condition="System.HasAddon(plugin.program.autocompletion)">Addon.OpenSettings(plugin.program.autocompletion)</onclick>
 				</control>
-				<control type="button" id="208">
+				<control type="button" id="209">
 					<width>1200</width>
 					<height>66</height>
-					<left>20</left>
 					<textwidth>1120</textwidth>
-					<label>$LOCALIZE[31083]</label>
+					<label> - $LOCALIZE[31083]</label>
 					<label2>$VAR[addon-imageselect]</label2>
 					<onclick condition="!System.HasAddon(script.image.resource.select)">InstallAddon(script.image.resource.select)</onclick>
 					<onclick condition="System.HasAddon(script.image.resource.select)">Addon.OpenSettings(script.image.resource.select)</onclick>
 				</control>
-				<!-- Supported -->
-				<control type="label" id="209">
-					<width>1220</width>
-					<height>62</height>
-					<font>Font33</font>
-					<align>left</align>
-					<label>[I]$LOCALIZE[31084][/I]</label>
-					<textcolor>$VAR[TextColor2]</textcolor>
-				</control>
-				<control type="button" id="210">
+				<!-- <control type="button" id="210">
 					<width>1200</width>
 					<height>66</height>
-					<left>20</left>
 					<textwidth>1120</textwidth>
-					<label>$LOCALIZE[31085]</label>
+					<label> - $LOCALIZE[31085]</label>
 					<label2>$VAR[addon-nextup]</label2>
 					<onclick condition="!System.HasAddon(service.nextup.notification)">InstallAddon(service.nextup.notification)</onclick>
 					<onclick condition="System.HasAddon(service.nextup.notification)">Addon.OpenSettings(service.nextup.notification)</onclick>
-				</control>
+				</control> -->
 				<control type="button" id="211">
 					<width>1200</width>
 					<height>66</height>
-					<left>20</left>
 					<textwidth>1120</textwidth>
-					<label>$LOCALIZE[31086]</label>
+					<label> - $LOCALIZE[31086]</label>
 					<label2>$VAR[addon-artistslideshow]</label2>
 					<onclick condition="!System.HasAddon(script.artistslideshow)">InstallAddon(script.artistslideshow)</onclick>
 					<onclick condition="System.HasAddon(script.artistslideshow)">Addon.OpenSettings(script.artistslideshow)</onclick>
